src/config/dynamic-theme-fixes.config

```diff
@@ -151,6 +151,18 @@ span[class^="skycon swip"]
 
 ================================
 
+app.datadoghq.com
```

Comment:
Please, preserve alphabetical order by URL. By the way, there is already `app.datadoghq.com` in the list.

Reply:
done